### DISCUSSION TITLE
Info per dispositivi Zwave

### DISCUSSION THREAD

**Discussion by effe-7** [indomus/forum#2672] [HUB: Home Assistant]
Ciao a tutti, sono nuovo e faccio i miei complimenti al canale indomus che ho trovato uno dei più completi.

Attualmente sto usando come hub HOMEY che ho trovato molto pratico e semplice nel gestirlo ma nella sua semplicità ha ovviamente delle limitazioni gestionali per questo vorrei provare HomaAssistant attualmente installato su windows

Parto dai dispositivi che ho per la maggiore che sono gestiti alcuni da hub Ikea correttamente configurato e poi la maggioranza sono dispostivi z-wave di Fibaro e Aotec per i quali ho preso gateway usb di aotec
Dopo questa lunga premessa è qui che nascono le mie domande...

Il pc che ospita l'hub è situato al piano terra dell'abitazione ed i dispositivi sono sparsi per casa fino al 2piano come faccio per associarli ad HomeAssistant?

Inoltre per questo periodo di prova/transizione potrei averli associati ad entrambi gli hub?

Anticipatamente GRAZIE

**Comment by indomus** [comment]
ciao e bentrovato,
puoi installarti Zwvejs2mqtt, associarli portandoli vicini all'antenna e poi riposizionarli dove previsto, se il segnale arriva. altrimenti ti servono dei repeater.
no: purtroppo non puoi abbinarli contemporaneamente a più bridge, questo è possibile farlo colo con Thread/Matter over- thread.

**Comment by effe-7** [comment]
Grazie per la solerte risposta.

Ho già dei repeter per allungare il segnale ma il mio problema è che installando homeassistant sul pc ed i dispositivi sono installati all'interno di prese ed interruttori sparsi per casa e non posso spostare il pc come posso fare, c'è un modo per associare i dispositivi senza avvicinarli al pc?
Grazie

**Reply by indomus** [reply]
non hai molta scelta se non provare ad associarli "da lontano" e sperare che la rete sia adeguata. ma solo provando, puoi capirlo.
